src/tests/functional/plugin/gna/shared_tests_instances/single_layer_tests/gru_sequence.cpp

```diff
@@ -0,0 +1,150 @@
+            manager.run_passes(function);
+            bool ti_found = is_tensor_iterator_exist(function);
+            EXPECT_EQ(ti_found, true);
+        } else {
+            bool ti_found = is_tensor_iterator_exist(function);
+            EXPECT_EQ(ti_found, false);
+        }
+    }
+
+    InferenceEngine::Blob::Ptr GenerateInput(const InferenceEngine::InputInfo &info) const override {
+        InferenceEngine::Blob::Ptr blob = make_blob_with_precision(info.getTensorDesc());
+        blob->allocate();
+        auto* rawBlobDataPtr = blob->buffer().as<float*>();
+        std::vector<float> values = CommonTestUtils::generate_float_numbers(blob->size(), -0.002f, 0.002f);
+        for (size_t i = 0; i < blob->size(); i++) {
+            rawBlobDataPtr[i] = values[i];
+        }
+        return blob;
+    }
+};
+
+TEST_P(GRUSequenceGNATest, CompareWithRefs) {
+        Run();
+}
+
+}  //  namespace LayerTestsDefinitions
+
+using namespace LayerTestsDefinitions;
+
+namespace {
+    std::vector<ngraph::helpers::SequenceTestsMode> mode{ngraph::helpers::SequenceTestsMode::CONVERT_TO_TI_MAX_SEQ_LEN_CONST,
+                                                         ngraph::helpers::SequenceTestsMode::PURE_SEQ};
+    // output values increase rapidly without clip, so use only seq_lengths = 1
+    std::vector<size_t> seq_lengths_zero_clip{1};
+    std::vector<size_t> seq_lengths_clip_non_zero{1};
+    std::vector<size_t> batch{1};
+    std::vector<size_t> hidden_size{1, 10};
+    // std::vector<size_t> input_size{10};
+    std::vector<std::vector<std::string>> activations = {{"relu", "tanh"}, {"tanh", "sigmoid"}, {"sigmoid", "tanh"},
+                                                         {"tanh", "relu"}};
+    std::vector<bool> linear_before_reset = {true, false};
+    std::vector<float> clip{0.f};
+    std::vector<float> clip_non_zeros{0.7f};
+    std::vector<ngraph::op::RecurrentSequenceDirection> direction = {ngraph::op::RecurrentSequenceDirection::FORWARD,
+                                                           ngraph::op::RecurrentSequenceDirection::REVERSE,
+                                                           ngraph::op::RecurrentSequenceDirection::BIDIRECTIONAL
+    };
+    std::vector<InferenceEngine::Precision> netPrecisions = {InferenceEngine::Precision::FP32,
+                                                             InferenceEngine::Precision::FP16};
+
+    INSTANTIATE_TEST_SUITE_P(smoke_GRUSequenceCommonZeroClip, GRUSequenceGNATest,
+                            ::testing::Combine(
+                                    ::testing::ValuesIn(mode),
+                                    ::testing::ValuesIn(seq_lengths_zero_clip),
+                                    ::testing::ValuesIn(batch),
+                                    ::testing::ValuesIn(hidden_size),
+                                    // ::testing::ValuesIn(input_size), // hardcoded to 10 due to Combine supports up to 10 args
```

Comment:
Maybe it makes sense s to combine the shape related parameters into some structure to avoid it?

Reply:
I didn't change this, since we use shared test params, and it requires making changes across all plugins, that used this params tuple. But I can`t change arm_plugin, which is in the control folder.

Do you think that we should introduce our own test parameters?

Reply:
@AntonGrishin ok, let's keep it as is for this PR